web_widget_slickroom/README.rst

```diff
@@ -0,0 +1,94 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===================================================
+Slick Carousel Widget with DarkroomJS Image Editing
+===================================================
+
+This module extends the `Slick`_ carousel widget provided by
+`web_widget_slick` to include the `DarkroomJS`_ image editing features provided
+by `web_widget_darkroom`.
+
+.. _Slick: http://kenwheeler.github.io/slick
+.. _DarkroomJS: https://github.com/MattKetmo/darkroomjs
+
+Usage
+=====
+
+To create a Slick carousel widget with DarkroomJS support, follow the
+usage instructions in the `web_widget_slick` documentation, but replace
+"one2many_slick_images" with "slickroom" in the field definition, as shown
+here::
+
+    <field name="image_ids" widget="slickroom" options="{}"/>
+
+To edit an image in a carousel, simply click the Edit button in the form view,
+then click on the image you wish to edit to open a DarkroomJS modal. Edit the
+image as desired according to the `web_widget_darkroom` documentation, and
+click Save to save the changes and update the carousel.
```

Comment:
Are m2m fields also supported? I would assume so, but there's no mention of this in the README and no m2m examples in the example module. Maybe add a note somewhere clearing this up.